contracts/core/extensions/CoreInternal.sol

```diff
@@ -104,6 +104,30 @@ contract CoreInternal is
         state.setTokens = state.setTokens.remove(_set);
     }
 
+    /**
+     * Adds or removes a price library to the mapping of tracked price libraries. Can only be set by
+     * owner of Core.
+     *
+     * @param  _priceLibrary   The address of the Price Library to enable or disable
+     */
+    function setPriceLibrary(
+        address _priceLibrary
+    )
+        external
+        onlyOwner
+    {
+        // Mark as true or false in validPriceLibraries mapping
+        state.validPriceLibraries[_priceLibrary] = !state.validPriceLibraries[_priceLibrary];
```

Comment:
Same, a ton of lookups there. I'll be using one of the interfaces above for the other methods.